src/Pages/Home/index.tsx

```diff
@@ -1,110 +1,51 @@
-                    className="d-block w-100"
-                    alt="..."
-                  />
-                  <div className="carousel-caption d-none d-md-block">
-                    <h5>First slide label</h5>
-                    <p>
-                      Some representative placeholder content for the first
-                      slide.
-                    </p>
-                  </div>
-                </div>
-                <div className="carousel-item">
-                  <img
-                    src="https://www.shutterstock.com/image-vector/janmashtami-festival-vector-lord-krishna-260nw-2178627335.jpg"
-                    className="d-block w-100"
-                    alt="..."
-                  />
-                  <div className="carousel-caption d-none d-md-block">
-                    <h5>Second slide label</h5>
-                    <p>
-                      Some representative placeholder content for the second
-                      slide.
-                    </p>
-                  </div>
-                </div>
-                <div className="carousel-item">
-                  <img
-                    src="https://www.shutterstock.com/image-illustration/scene-battlefield-mahabharata-krishna-arjuna-600w-2091772597.jpg"
-                    className="d-block w-100"
-                    alt="..."
-                  />
-                  <div className="carousel-caption d-none d-md-block">
-                    <h5>Third slide label</h5>
-                    <p>
-                      Some representative placeholder content for the third
-                      slide.
-                    </p>
-                  </div>
-                </div>
-              </div>
-              <button
-                className="carousel-control-prev"
-                type="button"
-                data-bs-target="#carouselExampleCaptions"
-                data-bs-slide="prev"
-              >
-                <span
-                  className="carousel-control-prev-icon"
-                  aria-hidden="true"
-                ></span>
-                <span className="visually-hidden">Previous</span>
-              </button>
-              <button
-                className="carousel-control-next"
-                type="button"
-                data-bs-target="#carouselExampleCaptions"
-                data-bs-slide="next"
-              >
-                <span
-                  className="carousel-control-next-icon"
-                  aria-hidden="true"
-                ></span>
-                <span className="visually-hidden">Next</span>
-              </button>
-            </div>
-          </StyledCarousel>
-        </div>
-      </Background>
+    <div>
+      <ReactCarousel />
+      <HeadingStyle>
+        <h1>ABOUT US</h1>
+        <Image src={TitlebaseUrl} />
+        <ImageStyle>
+          <img src={about} className="rounded" />
+        </ImageStyle>
+        <TextStyle> {mapAboutUs}</TextStyle>
```

Comment:
```suggestion
        <TextStyle> {aboutUs.map(paragraph => <p>{paragraph}</p>)}</TextStyle>
```

Apply this everywhere and remove code line 8 to 14

Reply:
Prettier automatically adds brackets to the  paragraph in the map function